pkg/v1/tkg/web/src/app/views/landing/aws-wizard/aws-wizard.component.ts

```diff
@@ -222,6 +222,21 @@ export class AwsWizardComponent extends WizardBaseDirective implements OnInit {
         return this.getFieldValue('awsNodeSettingForm', 'clusterName');
     }
 
+    /**
+     * @method to return cli command string according to special selection
```

Comment:
please add the method name after @method. Great job with documenting the method though, much appreciated.